musicround/helpers/spotify_client_manager.py

```
@@ -0,0 +1,83 @@
+"""
+Updated core.py with improved Spotify OAuth handling
+"""
+import logging
+import datetime
+from flask import current_app, redirect, url_for, flash, session
```

Ruff (0.11.9)

`flask.current_app` imported but unused

Remove unused import

(F401)